### PR TITLE
Delete AutoPiTaskService, which isn't used anywhere anymore once /upd…

### PR DESCRIPTION
**Pull request by elffjs**
…ate is gone

I wish this meant that we could kill Redis but alas.

We can, however, get rid of our forked version of taskq.

### CODE REVIEW

**Comment by zer0stars** [comment]
I thought we used the autopi task service to set the template?

**Comment by elffjs** [comment]
@JamesReate Giving this a nudge.

**Review by JamesReate — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -46,7 +47,6 @@ require (
 	github.com/swaggo/swag v1.16.3
 	github.com/testcontainers/testcontainers-go v0.27.0
 	github.com/tidwall/gjson v1.17.0
-	github.com/vmihailenco/taskq/v3 v3.2.8
```
Comment (JamesReate) [inline go.mod:49]: yay

**Comment by JamesReate** [comment]
yeah this is good to remove - was only used to apply the update and keep track of that.

**Pull request merged**